image/php-fpm/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu
+
+# ADD /laravel /var/www
```

Comment:
You'll notice in `image/php-fpm/Dockerfile`, `image/cli/Dockerfile` and `image/nginx/Dockerfile`, I've commented out the `ADD` instruction. This is a bit tricky. We want all these images to share the `laravel` directory from the project root, but you can't add files outside the current build context. See https://github.com/docker/docker/issues/2745 for more info on this.